composer.json

```diff
@@ -55,5 +55,10 @@
         "sort-packages": true
     },
     "minimum-stability": "dev",
-    "prefer-stable": true
+    "prefer-stable": true,
+    "scripts": {
+        "post-update-cmd": [
+            "@php artisan horizon:assets"
```

Comment:
How about creating a class that would check whether `artisan` exists and if so, it would run `Artisan::call('horizon:assets')`?

Reply:
[Nevermind](https://github.com/laravel/horizon/issues/554#issuecomment-529456544)